contracts/DixelClubV2NFT.sol

```diff
@@ -25,7 +25,7 @@ contract DixelClubV2NFT is ERC721Enumerable, Ownable, SVGGenerator {
     uint256 private _tokenIdTracker;
     IDixelClubV2Factory private _factory;
 
-    uint40 private _initializedAt;
+    uint32 private _initializedAt;
```

Comment:
uint32 supports up to Feb 2106, and that's only 84 years from now.. I guess I could live up to 300 years old, so I want this app to be running at least until I die.. 😛

anyway, I guess this doesn't save any gas here because it takes one minimum storage there (32 bytes).

Reply:
Haha, this appears to be used only for whether the contract has been initialized.
Metadata has 4 bits of space left. You can use this slot to save more storage space!